### ISSUE TITLE
Error when resetting FPP settings

### ISSUE THREAD

**Issue by FlorianLeChat** [FPtje/DarkRP#2815]
[//]: # (DO NOT REMOVE THIS TEMPLATE.)
[//]: # (GITHUB ISSUES ARE **NOT** FOR HELP)
[//]: # (GO TO THE FORUMS INSTEAD: https://forum.darkrp.com/ )

### Description of the bug
[//]: # (Describe the issue as accurately as possible)

When I execute the "[FPP_ResetAllSettings](https://github.com/FPtje/DarkRP/blob/2c60716999c1ce8a6356eea5f62990fe07f22812/gamemode/modules/fpp/pp/server/settings.lua#L884-L903)" command, it creates an error in the server console. Tested on a dedicated server without addon and with the latest version of the DarkRP of the github. 

### How to make the bug happen
[//]: # (Knowing how to make a bug happen allows the developers to figure out)
[//]: # (what the cause of the problem is and whether a certain fix solves it)

Run the "FPP_ResetAllSettings" command in the client console (F10) or in the server console.

### Lua errors
[//]: # (Note that errors on server startup are more important than other ones)
[//]: # (because they can cause other errors. Always look in the startup log of the server!)

L 11/13/2018 - 19:17:08: Lua Error: Couldn't include file 'fpp/sh_settings.lua' (File not found) (@gamemodes/darkrp/gamemode/modules/fpp/pp/server/settings.lua (line 896))

Couldn't include file 'fpp/sh_settings.lua' (File not found) (@gamemodes/darkrp/gamemode/modules/fpp/pp/server/settings.lua (line 896))
L 11/13/2018 - 19:17:08: Lua Error: lua/includes/extensions/table.lua:484: bad argument #1 to 'pairs' (table expected, got nil)

lua/includes/extensions/table.lua:484: bad argument #1 to 'pairs' (table expected, got nil)
L 11/13/2018 - 19:17:09: Lua Error: Warning! A net message is already started! Discarding...

Warning! A net message is already started! Discarding...
L 11/13/2018 - 19:19:41: Lua Error:
[ERROR] gamemodes/darkrp/gamemode/modules/fpp/pp/server/core.lua:292: attempt to index field 'Settings' (a nil value)
  1. v - gamemodes/darkrp/gamemode/modules/fpp/pp/server/core.lua:292
   2. unknown - lua/includes/modules/hook.lua:84

[ERROR] gamemodes/darkrp/gamemode/modules/fpp/pp/server/core.lua:292: attempt to index field 'Settings' (a nil value)
  1. v - gamemodes/darkrp/gamemode/modules/fpp/pp/server/core.lua:292
   2. unknown - lua/includes/modules/hook.lua:84``

### Why the developer of DarkRP is responsible for this issue
[//]: # (By posting on Github, you ask the developers of DarkRP to solve the problem)
[//]: # (It may seem obvious to you, but you have to make clear why they are the right people to look at it)

Because FPP is installed by default in the DarkRP gamemode.

**Note : I used MySQL on the DarkRP when I tested this.**

**Comment by Bo98** [comment]
[This](
https://github.com/FPtje/DarkRP/blob/2c60716999c1ce8a6356eea5f62990fe07f22812/gamemode/modules/fpp/pp/server/settings.lua#L896) looks like it won’t work with FPP in DarkRP’s directory structure.

**Comment by FlorianLeChat** [comment]
I confirm the following access path on my server : garrysmod/gamemodes/darkrp/gamemode/modules/fpp/pp/server/core.lua

I downloaded the DarkRP gamemode on this github to my computer and sent it to my Garry's Mod server via FTP with Filezilla. A few weeks ago, I had many problems when Filezilla sent files to my server, I noticed that a bug was present but since then it has been fixed. In doubt, I just tried again with another software, WinSCP but I still have the same problem, then I tried directly in singleplayer but I have the [same error](https://image.noelshack.com/fichiers/2018/46/3/1542212608-error.png) in the console when executing the command.

Finally I tried in sandbox with the [FPP addon](https://github.com/FPtje/Falcos-Prop-protection) and I [don't have the error](https://image.noelshack.com/fichiers/2018/46/3/1542213030-sandbox.png) when I execute the command, so I agree with Bo98's opinion.
